cmd/contact-exporter/main.go

```diff
@@ -52,6 +53,33 @@ func (c contactExporter) findContacts() ([]contact, error) {
 	return contactsList, nil
 }
 
+func (c contactExporter) findContactsForDomains(domains []string) ([]contact, error) {
+	var contactsList []contact
+	for _, domain := range domains {
+		// Pass the same list in each time, gorp will happily just append to the slice
+		// instead of overwriting it each time
+		// https://github.com/coopernurse/gorp/blob/9cd2b5ef5b82fde4e7c51776ac3f94398b8af076/gorp.go#L1644-L1651
```

Comment:
Small nit: Why is this pointed at a blob on a fork of https://github.com/go-gorp/gorp instead of the upstream?